### PR TITLE
Dialog changes

### PR DESCRIPTION
**Pull request by xtpclark**
It's a little broken around the deploying the site template to /var/www/dev/ .  I was thinking it should be something like /var/www/prodiem/{dev,stage,prod} vs /var/www/dev/prodiem, /var/www/stage/prodiem 
... I think /var/www/CUSTOMER/{dev,stage,prod} is easier to keep track of.

### CODE REVIEW

**Comment by bixwilson** [comment]
@xtpclark /var/www/[customer]/dev makes sense to me. 2 questions: 
1. Did @amikheychik get a chance to offer an opinion? 
2. Did you already make that change, or you're just considering it?

**Comment by amikheychik** [comment]
@bixwilson `/var/www/[customer]/[environment]` seems flexible enough, I'm not sure though we want all customers' production on the same box. We had this long time ago (in a universe far away) back in Artsmith, and was a bad idea, since when server is down everyone is unhappy. But for `dev`/`stage` environments that would be optimal.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,20 @@
 *.backup
 *.md5sum
 cleansockets.sh
+.already_ran_update
+phpunit.phar.1
```
Comment (amikheychik) [inline .gitignore]: this `.1` doesn't look right

**Comment by gilmoskowitz** [comment]
superseded by other work

**Pull request closed without merging**